cilantro_audit/widgets/view_audit_templates.kv

```diff
@@ -0,0 +1,122 @@
+#:kivy 1.11.1
+
+
+Manager:
+    RootPage:
+    AnotherPage:
```

Comment:
I don't think the Kivy version is needed as long as we specify it in the Python code.
For the screen manager stuff, the `root` variable will already have the `manager` property once we start
the application from `HomePage`, thus this can be removed as well. If you guys need help hooking it up, I'll be happy to help.

Reply:
Well, for this page, there is also no need to use it from Python. This make it a bit cleaner since the Kivy app comes with an attribute 'root' that looks for a root widget, and that is the root widget (screen manager) inside the .kv file. I think this way is cleaner.

Reply:
Well, I added the version because that's the standard required by Kivy:
"The content of the file should **always** start with the Kivy header, where version must be replaced with the Kivy language version you’re using."
[source](https://kivy.org/doc/stable/api-kivy.lang.html#syntax-of-a-kv-file)

About The Screen Manager, Kivy app include an attribute `root` that looks for a root widget within a kv file (or a kv string) that has the same app name and is in the same folder. However, since in our case the kv files are in a different directory, the app.root attribute gets never initialized until we explicitly load the file into our app. Luckily, the load function comes with the app (no need for a builder) and what it basically does, it looks for a root widget within the kv file we loaded. So, instead of importing ScreenManager() in Python (which does the same thing, I thought it's cleaner to add it to the kv file as a root widget.